packages/fast-usdc/src/types.ts

```diff
@@ -70,7 +70,7 @@ export interface CctpTxEvidence {
   txHash: EvmHash;
 }
 
-export interface EvidenceWithRisk {
+export interface EvidenceWithRisk extends CopyRecord {
```

Comment:
why?

Reply:
to meet the `Passable` constraint on things that go across the membrane.